pkg/systemlogmonitor/log_monitor.go

```diff
@@ -196,6 +233,19 @@ func initialConditions(defaults []types.Condition) []types.Condition {
 	return conditions
 }
 
+func initialProblemGauges(initialConditions []types.Condition) {
```

Comment:
Shall this be removed as we have `initializeProblemMetricsOrDie` now?

Reply:
Sounds fair. Let's remove it.

Reply:
This function is only useful for [this](https://github.com/kubernetes/node-problem-detector/issues/304#issuecomment-508870052).

When I work on #304 , I will take these into consideration.